src/Translator.php

```diff
@@ -397,6 +448,42 @@ private function mergeJson(): void
         $this->log('Finished merging ' . count($this->originalJson) . ' json files');
     }
 
+    /**
+     * Transifex may add english strings to translation json files, remove them
+     */
+    private function removeEnglishStringsFromJsonTranslations(): void
+    {
+        $this->log('Removing english from json translation files');
+        $enPath = '';
+        $enJson = null;
+        $count = 0;
+        foreach (array_keys($this->originalJson) as $path) {
+            if (!file_exists($path)) {
+                continue;
+            }
+            if (!$enPath) {
+                $enPath = preg_replace('#/[^/\.]+\.json$#', '/en.json', $path);
+            }
+            if (!$enJson) {
+                $enJson = $this->jsonDecode(file_get_contents($enPath));
+            }
+            if ($enPath === $path) {
+                continue;
+            }
+            // Remove any keys where the value is the same as the source english value
+            $contentJson = $this->jsonDecode(file_get_contents($path));
```

Comment:
```suggestion
                $enJson = $this->jsonDecode(file_get_contents($enPath), true);
            }
            if ($enPath === $path) {
                continue;
            }
            // Remove any keys where the value is the same as the source english value
            $contentJson = $this->jsonDecode(file_get_contents($path), true);
```

Explicitly use associative arrays - otherwise it could be objects depending on what the `JSON_OBJECT_AS_ARRAY` flag is set to.

Reply:
Nevermind, I forgot we abstracted that.